spec/abilities/event_ability_spec.rb

```diff
@@ -822,4 +822,24 @@
     end
   end
 
+  context 'as person without roles' do
+    let(:person_without_roles) do
+      person = Fabricate(Person.name.downcase.to_sym)
```

Comment:
```suggestion
      person = Fabricate(:person)
```
for the Fabricator it's pretty ok to just use symbols.